### PR TITLE
Fix getting started template with IRSA

### PR DESCRIPTION
**Pull request by JasonXZLiu**
**Description:**

Change the service account to be the one created when setting up the IAM Roles for service accounts tutorial. This IAM Role will be attached with the proper permissions needed.

### CODE REVIEW

File: examples/eks/prometheus-pipeline/eks-prometheus-sidecar.yaml [hunk 1]
```diff
@@ -71,8 +71,10 @@ data:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: adotcol-admin
+  name: amp-iamproxy-ingest-service-account
   namespace: adot-col
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::YOUR_ACCOUNT_ID:role/amp-iamproxy-ingest-role
```
Comment (mxiamxia) [inline examples/eks/prometheus-pipeline/eks-prometheus-sidecar.yaml:77]: Can we use `<>` for the variables in the template?

**Comment by mxiamxia** [comment]
We named the example file `examples/eks/prometheus-pipeline/eks-prometheus-sidecar.yaml`. But in the template,  we deploy the collector as service from DaemonSet mode which is not fitting the meaning of the file name. This example seems misleading.

File: examples/eks/prometheus-pipeline/eks-prometheus-sidecar.yaml (hunk 1 above)
Comment (JasonXZLiu) [reply examples/eks/prometheus-pipeline/eks-prometheus-sidecar.yaml:77]: I'll change that throughout our template

**Review by mxiamxia — APPROVED** [review]
Thanks!

**Pull request merged**